host_core/lib/host_core/actors/actor_supervisor.ex

```diff
@@ -28,11 +29,12 @@ defmodule HostCore.Actors.ActorSupervisor do
   @spec start_actor(
           bytes :: binary(),
           host_id :: String.t(),
+          lattice_prefix :: String.t(),
           oci :: String.t(),
           count :: integer(),
           annotations :: map()
         ) :: {:error, any} | {:ok, [pid()]}
-  def start_actor(bytes, host_id, oci \\ "", count \\ 1, annotations \\ %{})
+  def start_actor(bytes, host_id, lattice_prefix, oci \\ "", count \\ 1, annotations \\ %{})
```

Comment:
That's right, but we can't look up `config` if we fail to lookup the host ID for whatever reason (could happen if this host receives a request for the wrong host ID)

Reply:
Hmm, I feel like passing an invalid host ID should result in an early nack vs a failed event... It looks like that's what we do when getting a host inventory: https://github.com/wasmCloud/wasmcloud-otp/blob/main/host_core/lib/host_core/control_interface/host_server.ex#L46

However, this particular error case shouldn't happen, since the HostServer subscribes to a host-ID-specific topic, and the server is supervised by the VirtualHost, so if the host stops/dies, the subscription to that topic should also stop, right?